ReleaseProcessAutomation.Tests/MsBuild/MsBuildInvokerTests.cs

```diff
@@ -59,15 +60,77 @@ public void InvokeMSBuildAndCommit_NoMSBuildPathSpecified_EndsEarly()
+      var testConsole = new TestConsole();
 
       var msBuildInvoker = new MSBuildCallAndCommit(gitClientStub.Object, _config, msBuildMock.Object,
-          _consoleStub.Object);
+          testConsole);
 
       var act = msBuildInvoker.CallMSBuildStepsAndCommit(MSBuildMode.PrepareNextVersion, version);
 
       Assert.That(act, Is.EqualTo(-1));
+      Assert.That(testConsole.Output, Does.Contain("There was no MSBuildPath specified in the config"));
+
       msBuildMock.Verify(n => n.CallMSBuild("", It.IsAny<string>()),Times.Never);
     }
+    
+    [Test]
+    public void InvokeMSBuildAndCommit_CorrectMSBuildPathSpecified_FinishesWithoutErrors()
+    {
+        
+        var version = new SemanticVersion
+                      {
+                              Major = 1,
+                              Minor = 1,
+                              Patch = 1,
+                      };
+
+        _config.MSBuildSettings.MSBuildPath = Path.Combine(TestContext.CurrentContext.TestDirectory, "msbuild.exe");
+        File.Delete(_config.MSBuildSettings.MSBuildPath);
+        var writer = File.Create(_config.MSBuildSettings.MSBuildPath);
+        writer.Close();
+        var gitClientStub = new Mock<IGitClient>();
+        gitClientStub.Setup(_ => _.IsWorkingDirectoryClean()).Returns(true);
+        var msBuildMock = new Mock<IMSBuild>();
+        
+        var testConsole = new TestConsole();
+
+        var msBuildInvoker = new MSBuildCallAndCommit(gitClientStub.Object, _config, msBuildMock.Object,
+                testConsole);
+
+        var act = msBuildInvoker.CallMSBuildStepsAndCommit(MSBuildMode.PrepareNextVersion, version);
+
+        Assert.That(act, Is.EqualTo(0));
+
+        msBuildMock.Verify(n => n.CallMSBuild(It.IsAny<string>(), It.IsAny<string>()));
+        File.Delete(_config.MSBuildSettings.MSBuildPath);
+    }
+    
+    [Test]
+    public void InvokeMSBuildAndCommit_WrongMSBuildPathSpecified_EndsEarlyWithProperMessage()
+    {
+        var version = new SemanticVersion
+                      {
+                              Major = 1,
+                              Minor = 1,
+                              Patch = 1,
+                      };
+
+        _config.MSBuildSettings.MSBuildPath = "DefinitelyNotAFolder/DefinitelyNotAnMSBuildPath.definitelyNotAnExe";
+        var gitClientStub = new Mock<IGitClient>();
+        var msBuildMock = new Mock<IMSBuild>();
+        msBuildMock.Setup(_ => _.CallMSBuild("", It.IsAny<string>())).Verifiable();
+
+        var testConsole = new TestConsole();
+
+        var msBuildInvoker = new MSBuildCallAndCommit(gitClientStub.Object, _config, msBuildMock.Object,
+                testConsole);
+
+        var act = msBuildInvoker.CallMSBuildStepsAndCommit(MSBuildMode.PrepareNextVersion, version);
+
+        Assert.That(act, Is.EqualTo(-1));
+        Assert.That(testConsole.Output.ReplaceLineEndings(""), Does.Contain($"The configured MSBuildPath '{_config.MSBuildSettings.MSBuildPath}' does not exist"));
```

Comment:
Ah okay. Yes. we need the absolute path in the output, otherwise errors due to a mis-matched Env.CurrentDirectory can drive you inside. (talking from experience here :) )